website/pages/en/index.js

```diff
@@ -201,6 +206,18 @@ class HomeSplash extends React.Component {
           </div>
           </div>
       </SplashContainer>
+      <div align="center">
+        <div align="left" style={SOreminder}>
```

Comment:
The style looks uninspired – should we get design input on this before we post it?

Reply:
The `width: 90%` is my least favorite part of these styles – the width of this SO disclaimer doesn't match the width of the content above, or anything else on the page. However, I noticed that the mobile styles on this page could use some deeper cleanup, and that makes me wonder if the `width: 90%` is fine for now.